packages/framework/src/HydeServiceProvider.php

```diff
@@ -120,7 +120,7 @@ protected function registerModuleServiceProviders(): void
         $this->app->register(Modules\DataCollections\DataCollectionServiceProvider::class);
 
         if (env('DUSK_ENABLED', false)) {
```

Comment:
This feels very hacky to me. Code for testing should not be in a production class. If using Dusk proves useful, it's better to create a package that bootstraps test providers if that is needed.

Reply:
This is also why codecov patch is failing.